### PR TITLE
firebase-push-notification

### PR DESCRIPTION
**Pull request by ashok-choudhary-0**

### CODE REVIEW

File: controller/postController.js [hunk 1]
```diff
@@ -72,6 +73,8 @@ const addCommentOnPost = async (req, res) => {
     const postData = await postRef.get();
     if (postData.exists) {
       await postRef.collection("comments").add({ userUid, comment, createdAt: new Date(), updatedAt: new Date() })
+      const postUserUid = postData.updatedBy
```
Comment (yuvrajVitaAccess) [inline controller/postController.js]: why updatedBy? why not createdBy? @ashok-choudhary-0

File: controller/postController.js [hunk 2]
```diff
@@ -118,4 +121,18 @@ const getSinglePost = async (req, res) => {
     res.status(500).send(err.message)
   }
 }
-module.exports = { createPost, tagUser, removeTagUser, allPosts, addCommentOnPost, deleteCommentOnPost, editCommentOnPost, getSinglePost }
\ No newline at end of file
+const sendPushNotification = async (req, res, commentedUserUid, postUserUid, postUid) => {
+  try {
+    const message = {
+      notification: {
+        body: `${commentedUserUid} commented on your post ${postUid}`,
```
Comment (yuvrajVitaAccess) [inline controller/postController.js]: should be name for user and post title @ashok-choudhary-0

File: controller/postController.js [hunk 3]
```diff
@@ -118,4 +121,18 @@ const getSinglePost = async (req, res) => {
...
+      },
+      token: postUserUid
```
Comment (yuvrajVitaAccess) [inline controller/postController.js]: is this working? when you send userId is token? @ashok-choudhary-0

**Review by yuvrajVitaAccess — CHANGES_REQUESTED** [review]

File: controller/postController.js (hunk 1 above)
Comment (ashok-choudhary-0) [reply controller/postController.js]: sir in assignment it is mentioned that fieldname will be updatedBy so thats why i use updatedBy  when user creates a new post

**Pull request merged**